test/functionalTest/cases/1048_subscription_cache/get_subscription_before_and_after_cache_refresh_status_in_db_older.test

```diff
@@ -81,7 +81,7 @@ brokerStart CB 0 IPv4 -subCacheIval 2 --cache
 # 01. GET /v2/subscriptions and see dynamic fields set A
 # 02. Set dynamic fields in DB to set B
 # 03. GET /v2/subscriptions and see dynamic fields set B although cache has not been synched yet
-# 04. Wait 2.5s for cache refresh and GET /v2/subscriptions again to check everything is the same except
+# 04. Wait 2.5s for cache refresh and GET /v2/subscriptions again to check everything is the same, except failsCounter (see NOTE)
```

Comment:
I can't find the NOTE in this document

Reply:
It is in the other file. Note the text above:

```
# Same as get_subscription_before_and_after_cache_refresh.test but status
# in dynamic fields set B has last changed 2020-01-01T09:00:00Z (1577869200)
# which is older than the one in dynamic field set A, so that later one
# wins and in step 03 and 04 we get "active" instead of "inactive"
```

I think is better not duplicate the NOTE here. Duplicated information is harder to maintain.

Reply:
Ok, NTC